wiki/ContributingCode.md

```diff
@@ -1,5 +1,5 @@
 Endless Sky needs contributions in a lot of different forms: artwork, story writing, beta testing, etc. I'm a professional C++ programmer, so the code is probably the area where I need the least help. But particularly if you have experience with modern OpenGL, I'd welcome your input!
 
-If you are interested in contributing changes to the source code, first have a look at the [style guide](https://endless-sky.github.io/styleguide/styleguide.xml). You will probably disagree on some of the style points. Your personal style is probably just as well-thought-out and elegant and professional-looking as mine. But, if the code becomes a jumble of everyone's personal styles, the end result will be ugly and inconsistent. (If you make a _really_ good case, I'm open to changing the style guide and going through all the code to make it match that change.)
+If you are interested in contributing changes to the source code, first have a look at the [style guide](https://github.com/endless-sky/endless-sky/wiki/C++-Style-Guide). You will probably disagree on some of the style points. Your personal style is probably just as well-thought-out and elegant and professional-looking as mine. But, if the code becomes a jumble of everyone's personal styles, the end result will be ugly and inconsistent. (If you make a _really_ good case, I'm open to changing the style guide and going through all the code to make it match that change.)
```

Comment:
Should this section still be written in first-person from MZ's perspective? At this point, MZ hasn't been around for quite some time, and I don't see any way that anyone would be willing to make any major changes to the styleguide these days considering the fact that we probably have something like 100k lines of code in ES now.